www/runtest.php

```diff
@@ -3092,7 +3092,14 @@ function CheckRateLimit($test, &$error) {
 
   if(!$passesMonthly) {
     $error = '<p>You\'ve reached the limit for logged-out tests this month, but don\'t worry! You can keep testing once you log in, which will give you access to other nice features like:</p>';
-    $error .= '<script>window["_gaq"] && window["_gaq"].push("_trackEvent", "Error", "RateLimit", "MonthlyLimitHit", "' . $test['ip'] . '");</script>';
+    $error .= '<script>';
+    $error .= 'var intervalId = setInterval(function () { ';
+    $error .= 'if(window["_gaq"]) { ';
+    $error .= 'clearInterval(intervalId);';
+    $error .= 'window["_gaq"].push("_trackEvent", "Error", "RateLimit", "MonthlyLimitHit");';
+    $error .= ' }';
+    $error .= ' }, 500);';
+    $error .= '</script>';
```

Comment:
Maybe a [HEREDOC syntax](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) looks nicer. By using the HTML identifier, I even get HTML syntax highlighting in VSCode.

```suggestion
    $error .= <<<HTML
<script>
    var intervalId = setInterval(function () {
        if(window["_gaq"]) {
            clearInterval(intervalId);
            window["_gaq"].push("_trackEvent", "Error", "RateLimit", "MonthlyLimitHit");
        }
    }, 500);
</script>
HTML;
```

Reply:
_great call_